packages/client-api-schema/src/types.ts

```diff
@@ -167,28 +162,43 @@ export type ChallengeChannelRequest = JsonRpcRequest<'ChallengeChannel', {channe
 export type ChallengeChannelResponse = JsonRpcResponse<ChannelResult>;
 
 // Budget
+
+export interface ChannelBudget {
+  channelId: string;
+  status: 'pending' | 'available';
```

Comment:
I think we decided that we didn't need the `status` - you can check the channel status to find this out.

Reply:
2f84c95